backend/controller/admin/local_client.go

```diff
@@ -45,19 +47,52 @@ func (s *diskSchemaRetriever) GetActiveSchema(ctx context.Context) (*schema.Sche
 		return nil, fmt.Errorf("could not discover modules: %w", err)
 	}
 
-	sch := &schema.Schema{}
+	moduleSchemas := make(chan either.Either[*schema.Module, error], len(modules))
+	defer close(moduleSchemas)
+
 	for _, m := range modules {
-		config := m.Abs()
-		schemaPath := config.Schema()
-		if r, ok := s.deployRoot.Get(); ok {
-			schemaPath = filepath.Join(r, m.Module, m.DeployDir, m.Schema())
-		}
+		go func() {
+			// Loading a plugin can be expensive. Is there a better way?
+			plugin, err := languageplugin.New(ctx, m.Language)
```

Comment:
This change slows down this logic a fair bit because now we need to create a language plugin per module just so we can figure out where the schema is saved.
I'm not sure of a good solution to this, other than having a central place FTL saves schemas (similar to the central `/.ftl/` external modules) so that having a ModuleConfig is not readed to read the saved schema for a module.